src/test/miniscript_tests.cpp

```diff
@@ -116,7 +116,9 @@ struct KeyConverter {
 //! Singleton instance of KeyConverter.
 const KeyConverter CONVERTER{};
 
-using miniscript::operator"" _mst; // NOLINT(misc-unused-using-decls)
```

Comment:
please squash this

Reply:
not sure how that ended up in the wrong commit. fixed now